### PR TITLE
Correctly handle random parameters appearing in the denominator

### LINKED ISSUE #3156: When dividing by random parameter, one obtains fixed, not random values
**Describe the bug**

When initializing neuron or synapse properties with a random parameter appearing as a denominator, NEST assigns fixed values.

**To Reproduce**
1. (Minimal) reproducing example
```python
import nest
n = nest.Create('iaf_psc_alpha', 10, {'V_m': 1 / nest.random.uniform(0, 1)})
print(n.V_m)
```

**Expected behavior**
Expected results: 10 different membrane potential values
Observed result: 10 identical membrane potential values

**Desktop/Environment (please complete the following information):**
 - NEST-Version: nest master @ 8dd4b52e4d7f92c03e479537a95bd57719e77939

**Additional context**

We have traced the error to the implementation of `__rtruediv__()` here: https://github.com/nest/nest-simulator/blob/1ddea6bd1bdf605ba199e2ca756ce5333e7326e7/pynest/nest/lib/hl_api_types.py#L1086-L1089

### PR DESCRIPTION
**Pull request by heplesser**
This PR fixes #3156 and ensures that expressions containing random parameters in the denominator are evaluated correctly. It also fixes a minor error in the SLI interface.

Developed in collaboration with Jannik Grundler.

### CODE REVIEW

**Review by ackurth — APPROVED** [review]
Looks good to me.
You mention in the message that you developed the PR in collaboration with Jannik Grundler.
Maybe do a [co-authored commit](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) (if he has a GitHub account) to give him also some credit in the commit history.

**Comment by heplesser** [comment]
> Looks good to me. You mention in the message that you developed the PR in collaboration with Jannik Grundler. Maybe do a [co-authored commit](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) (if he has a GitHub account) to give him also some credit in the commit history.

Good point @ackurth, you can already find it here: https://github.com/nest/nest-simulator/pull/3158/commits

**Comment by ackurth** [comment]
> > Looks good to me. You mention in the message that you developed the PR in collaboration with Jannik Grundler. Maybe do a [co-authored commit](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) (if he has a GitHub account) to give him also some credit in the commit history.
> 
> Good point @ackurth, you can already find it here: https://github.com/nest/nest-simulator/pull/3158/commits

Ah, I did not see this.
But there is no GitHub account linked to this mail address.
Jannik's account name is `Jannik-Grundler`, the mail address is `jannik.grundler@rwth-aachen.de`.

File: testsuite/pytests/test_regression_issue-3156.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        pass  # operation not supported for parameter
+    else:
+        assert len(set(n.V_m)) == num_neurons
+
+
+def test_random_numer_and_denom():
+    """
+    For random parameters in numerator and denominator, we make the denominator uniform
+    on the set {-1, +1}. For 50 neurons we then have at least one positive and at least
+    one negative result with p = 2 * 2^-50 ≈ 2e-15.
```
Comment (mlober) [inline testsuite/pytests/test_regression_issue-3156.py:72]: Probability to have at least one positive and at least one negative result is 1-p, not p. Comment must be changed.

**Review by mlober — CHANGES_REQUESTED** [review]

File: testsuite/pytests/test_regression_issue-3156.py (hunk 1 above)
Comment (heplesser) [reply testsuite/pytests/test_regression_issue-3156.py]: Thanks for spotting this. I have reworded the comment now.

**Comment by heplesser** [comment]
> > > Looks good to me. You mention in the message that you developed the PR in collaboration with Jannik Grundler. Maybe do a [co-authored commit](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) (if he has a GitHub account) to give him also some credit in the commit history.
> > 
> > 
> > Good point @ackurth, you can already find it here: https://github.com/nest/nest-simulator/pull/3158/commits
> 
> Ah, I did not see this. But there is no GitHub account linked to this mail address. Jannik's account name is `Jannik-Grundler`, the mail address is `jannik.grundler@rwth-aachen.de`.

Fixed.

**Review by mlober — APPROVED** [review]
Thanks. Now this looks good to me.

**Pull request merged**